src/Http/Http.Extensions/test/RequestDelegateGenerator/RequestDelegateGeneratorTests.JsonBody.cs

```diff
@@ -0,0 +1,151 @@
+// Licensed to the .NET Foundation under one or more agreements.
+// The .NET Foundation licenses this file to you under the MIT license.
+using System.Globalization;
+using System.Text.Json;
+using Microsoft.AspNetCore.Http.Features;
+namespace Microsoft.AspNetCore.Http.Generators.Tests;
+
+public abstract partial class RequestDelegateGeneratorTests
```

Comment:
I think renaming this class would reduce confusion. Perhaps `RequestDelegateCreationTests`?

(not urgent: can be in its own PR in the future)

Reply:
Naming is hard! Thanks for the recommendation. I'll admit "Strategy" was probably too posh 😛  I went ahead and renamed it.